core/store/src/archive/cold_storage.rs

```diff
@@ -189,9 +195,10 @@ fn copy_state_from_store(
 
         let Some(trie_changes) = trie_changes else { continue };
         total_keys += trie_changes.insertions().len();
+        let mapped_shard_uid_key = get_shard_uid_mapping(hot_store, shard_uid).to_bytes();
         for op in trie_changes.insertions() {
-            // TODO(reshardingV3) Handle shard_uid not mapped there
-            let key = join_two_keys(&shard_uid_key, op.hash().as_bytes());
+            // TODO(reshardingV3) Test it properly. Currently this path is not triggered in testloop.
```

Comment:
nit: TODO(resharding) is the convention

Reply:
I was using TODO(reshardingV3) for a few months to differentiate it from old todos for previous reshardingV2. Will replace all of them with TODO(resharding) now.